dags/process_web_log.py

```diff
@@ -48,6 +51,36 @@ def load_data_func():
         tar.add(file_transform, arcname=os.path.basename(file_transform))
 
 
+def send_message_func():
+    # Email configuration
+    sender_email = "jclozdibil27@gmail.com"
+    receiver_email = "jose.lozano.dibildox@ulb.be"
+    password = "wuyd dfsc ntue tyix"  # App password

```

Comment:
is this a disposable mail? you may not want to disclose your pwd in a public repo